CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## `transcriptic` Changelog
 
+## v7.1.2
+Fixed
+- py27 tox validation
+
+## v7.1.1
+Added 
+- "--test_inputs" flag added to the transcriptic cli "launch" command. Adding new flag will modify the quick launch inputs into usable preview parameters for protocol debugging.
+
+## v7.0.1
+Fixed
+- Removal of `can_submit_autoprotocol` feature group in the default `.transcriptic`
+
```

Comment:
hmm these should really all be piecemeal, because we need to push/deploy these packages that we list as potential versions

Reply:
That is reasonable. I can work on chunking the updates.